Hero video and images

danielfmiranda opened the pull request:
Closes #6675 

**Link to sample test pages**:
Test Page with Video Header: https://foundation-s-hero-video-qlhqlb.herokuapp.com/en/blog/initial-test-blog-post-with-fixed-title/
Test Page with Image Header: https://foundation-s-hero-video-qlhqlb.herokuapp.com/en/blog/call-budget-yard-impact-art-trial/
Test Page with Neither: https://foundation-s-hero-video-qlhqlb.herokuapp.com/en/blog/test-glass-oil-book-color-opportunity/


**Steps to Test**:

1. Visit https://foundation-s-hero-video-qlhqlb.herokuapp.com/en/blog/initial-test-blog-post-with-fixed-title/, video should load at the top and start playing immediately.
2. Video should be responsive, and play/pause button should work as expected.
3. Visit https://foundation-s-hero-video-qlhqlb.herokuapp.com/en/blog/call-budget-yard-impact-art-trial/, there should be a responsive image at the top of the page. 
4. Visit https://foundation-s-hero-video-qlhqlb.herokuapp.com/en/blog/test-glass-oil-book-color-opportunity/, it should look like previous blog posts.


**NOTE:** Once PR is approved, we will need to add vimeo to the allowed media sources in heroku.
## Checklist

**Changes in Models:**
- [x] [Are my changes backward-compatible]

sabrinang reviewed (changes requested):
Responsive images and video work on the pages as well as the stop/play video feature.

I think the height shouldn't be taller than the home's hero and if they match these hero banner assets could be reusable between them and it gives one less different dimension to consider.
<img width="1683" alt="Screen Shot 2021-07-15 at 11 23 06 AM" src="https://user-images.githubusercontent.com/2492510/125838441-e8150e41-561a-4ee2-913a-06aabbd19e2f.png">

In the CMS, should there be some validation feedback here if both are entered or it could be formatted in a way that it is clear you can do either video or image but not both. Currently, if the editor fills in both fields the video shows. 
<img width="1133" alt="image" src="https://user-images.githubusercontent.com/2492510/125839051-16c52cd8-5adf-4182-8d08-3d25cd4a0b3c.png">

danielfmiranda commented:
Hi @sabrinang! I have updated the size of the hero video/image to match the size of the banner on the homepage, as well as updated the layout of the CMS fields and added some validation. They are now both under a "hero video/image" section, and it will not let users save the page with a video and image set, just one or the other. 

These changes are now live and ready for re-review, thanks!

sabrinang reviewed (approved):
Thanks for making the updates to the height and layout in the CMS Daniel, looks good.

KalobTaulien commented on network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/blog_page_heroguts.html:
```diff
@@ -0,0 +1,36 @@
+{% load wagtailcore_tags wagtailimages_tags i18n static%}
```
```suggestion
{% load wagtailimages_tags %}
```

KalobTaulien reviewed (approved):
Approving. One small commit suggestion (but please 2x check it first)

danielfmiranda commented:
@KalobTaulien thanks for the suggestion! I have tested it out on my local machine and can confirm that it brings no undesired side effects, so I have made those changes and commited them to the repo.

 I also found that this repo was experiencing the pause button shifting issue that you mentioned in #7103, so I have added the fix to this PR as well.
